cfgov/unprocessed/css/organisms/read-more.less

```diff
@@ -0,0 +1,95 @@
+/* topdoc
+  name: Expandable read more modifier
+  family: cf-organisms
+  notes:
+    - "Styles a one-way Expandable organism that previews content on mobile."
+  patterns:
+  - name: Expandable organism
+    markup: |
+      <div class="o-expandable o-expandable__read-more" data-read-more="true">
+          <div class="o-expandable_content" aria-expanded="false">
+              <div class="o-expandable_content-animated"></div>
+          </div>
+          <a class="o-expandable_target jump-link o-expandable_link" title="Expand content">
+              <span class="jump-link_text">
+                  <span class="o-expandable_cue o-expandable_cue-open">
+                      Read full answer
+                      <span class="cf-icon cf-icon-plus-round"></span>
+                  </span>
+                  <span class="o-expandable_cue o-expandable_cue-close">
+                      Hide full answer
+                      <span class="cf-icon cf-icon-minus-round"></span>
+                  </span>
+              </span>
+          </a>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        .o-expandable
+          .o-expandable_content
+            .o-expandable_content-animated
+          .o-expandable_target.o-expandable_link
+            .o-expandable_cue.o-expandable_cue-open
+            .o-expandable_cue.o-expandable_cue-close
+
+    notes:
+      - "One-way expandable. Displays an approximately 4-line 
+         preview of content on mobile screen sizes only. "
+  - name: .o-expandable__read-more (modifier)
+  tags:
+  - cf-organisms
+*/
+.o-expandable__read-more {
+    .o-expandable_content {
+        border-bottom: 0;
+        position: relative;
+    }
+    
+    .o-expandable_target {
+        display: none;
+    }
+
+    .o-expandable_content,
+    .o-expandable_content-animated {
+        height: auto;
+        padding: 0;
+        border-width: 0 !important;
+    }
+}
+
+.js .o-expandable__read-more[data-js-hook]:not(.o-expandable__expanded) {
+    .respond-to-max( @bp-xs-max, {
+        .o-expandable_content {
+            // shows approx. 4 lines of content
+            height: 88px; 
+            overflow: hidden;
+        }
+
+        .o-expandable_content:after {
+            // fades content out over approx. 2 lines
+            width: 100%; 
+            height: 44px; 
```

Comment:
The height is just an approximation since the actual content could contain headers or other elements with non-standard line-height and padding. It's based on the `@base-line-height` variable, though, so I could use that instead to calculate the values.